25_exceptions_lab/03.py

```diff
@@ -1,4 +1,31 @@
 """
 Write code to make the following unit test pass
 """
-
+import re
+import unittest
+
+class InvalidImageExt(Exception):
+    def __init__(self):
+        self.message = "Not an image extension"
+
+class ImageFile(object):
+    def __init__(self,file_name):
+        file_name = file_name.strip()
+        ext = re.search(r'\.(\w+)$', file_name)

```

Comment:
`os.path.splitetx` is a better way to get a file's extension than using a regexp (usually faster and more accurate)